.gitignore

```diff
@@ -1,8 +1,35 @@
+# Copying and distribution of this file, with or without modification,
```

Comment:
If you would like to put a license, I would prefer the same license as is in the other files of the project, namely the LGPLv2